src/lib/layer.js

```diff
@@ -446,7 +446,10 @@ export default class Layer {
     settings.polygonOffset = offsets;
 
     // Call subclass lifecycle method
-    this.draw({moduleParameters, uniforms, settings});
+    withParameters(this.context.gl,
```

Comment:
polygonOffset was a feature that layers used to get for free. Do we have a good reason to ask layers to do more work in their `draw` methods now?